data/json/mapgen/basement/basement_game.json

```diff
@@ -60,6 +60,9 @@
         "s": "f_sofa",
         "t": "f_table"
       },
+      "toilets": {
+        "&": {}
```

Comment:
Since you specify the toilet symbol here, it doesn't need to be specified above (in the `furniture` block) anymore.

Also, in the `terrain` block, there's no `&` symbol, so it uses the default terrain- solid rock. It's probably supposed to be rock floor.